ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/BlockingQueryPublisher.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 Confluent Inc.
+ * Copyright 2022 Confluent Inc.
```

Comment:
nit: unnecessary change

Reply:
You should when creating new classes. This is not a new class though?

Reply:
Looks like this change is still present in the latest version. Not a big deal, but hopefully all the other changes you meant to push in the latest revision made it in?